VAMobile/src/api/facilities/getFacilitiesInfo.tsx

```diff
@@ -0,0 +1,31 @@
+import { useQuery } from '@tanstack/react-query'
+
+import { FacilitiesPayload, Facility } from 'api/types/FacilityData'
+import { facilitiesKeys } from './queryKeys'
+import { get } from 'store/api'
+
+/**
+ * Fetch user facilities info
+ */
+
+export const getFacilitiesInfo = async (): Promise<Array<Facility> | undefined> => {
```

Comment:
If this function is only used in this file, there's no need to export it

Reply:
updated